packages/core/src/components/TextField/TextField.module.scss

```diff
@@ -111,6 +111,23 @@
   border-radius: 0 !important;
 }
 
+.textField .inputWrapper .tooltipContainer {
+  position: absolute;
+  top: 50%;
+  right: 0;
+  width: 32px;
+  height: 20px;
+  transform: translateY(-50%);
+}
```

Comment:
🙃
We'll deprecate this TextField one day 🤞🏼

Reply:
Hopefully sooner than later